### ISSUE TITLE
constants.PAD 

### ISSUE THREAD

**Issue by KeyBoarder8** [boathit/research-papers#1]
Why I can't use constants?It shows "AttributeError: module 'constants' has no attribute 'PAD'".Any other profile exits?

**Comment by boathit** [comment]
You are right, I missed the `constants.py` file in the previous commit and it has been added.

**Comment by KeyBoarder8** [comment]
@boathit Thank you so much!
